config/config.exs

```diff
@@ -33,6 +33,11 @@ config :antikythera, [
     prod: "antikythera.example.com"   ,
   ],
 
+  # HTTP header to write to gear log. The header must be specified in lower case.
```

Comment:
- We use these HTTP headers for writing something to gear log.
- We write these HTTP headers (and their values) to gear log.

I thought the current comment implies the former, but actual implementation is the latter.

```suggestion
  # HTTP headers to be written to gear log. The headers must be specified in lower case.
```